integration/admin-client/src/main/java/org/keycloak/admin/client/resource/ScopePermissionsResource.java

```diff
@@ -40,6 +34,17 @@
     Response create(ScopePermissionRepresentation representation);
 
     @Path("{id}")
+    @PUT
```

Comment:
These changes are not needed because you have these same methods in `org.keycloak.admin.client.resource.ScopePermissionResource`.